acceptance/suites/tests/code_commands/code_scripts.rb

```diff
@@ -0,0 +1,181 @@
+require 'json'
+
+confine :except, :type => 'pe'
+
+test_name 'SERVER-1118: Validate code-id-command feature in FOSS'
+
+git_repo='/git/puppetcode' #AKA "Control Repo" &| "Git Server" &| "origin"
+git_local_repo='/tmp/git'
+hostname = on(master, 'facter hostname').stdout.strip
+fqdn = on(master, 'facter fqdn').stdout.strip
+
+teardown do
+  on(master, 'rm -rf /root/.ssh/gittest_rsa*', :accept_all_exit_codes => true)
+  on(master, 'puppet resource user git ensure=absent')
+  on(master, "rm -rf #{git_repo}", :accept_all_exit_codes => true)
+  on(master, "rm -rf #{git_local_repo}", :accept_all_exit_codes => true)
+  on(master, "rm -rf /home/git/.ssh/authorized_keys", :accept_all_exit_codes => true)
+  #uninstall r10k
+  on(master, "/opt/puppetlabs/puppet/bin/gem uninstall r10k")
+  #return /etc/puppetlabs/code to original state
+  on(master, "rm -rf /etc/puppetlabs/code")
+  on(master, "puppet resource file /etc/puppetlabs/code ensure=directory")
+  #remove code_* scripts.
+  on(master, "rm -rf /opt/puppetlabs/server/apps/puppetserver/code-id-command_script.sh")
+  on(master, "rm -rf /opt/puppetlabs/server/apps/puppetserver/code_content_script.sh")
+end
+
+step "SETUP: Generate a new ssh key for the root user account to use with the git server"
```

Comment:
Nit: Like to see consistent use of single- and double-quotes. Most of the other `step` statements use single-quotes.